src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
```

Comment:
-missmatch between variable name and type, lottery is not a ball, both classes exist separately
-it is not necessary to create a new lottery each time, create one and reuse it